WordPress/src/androidTest/java/org/wordpress/android/e2e/ContactUsTests.java

```diff
@@ -21,11 +18,6 @@ public void setUp() {
         logoutIfNecessary();
     }
 
-    @After
-    public void tearDown() {
-        pressBackUntilElementIsDisplayed(R.id.continue_with_wpcom_button);
-    }
-
```

Comment:
Removing teardown completely because it does not look necessary here (these tests do not log in, and the app restarts before each test, which ensures the same state regardless of where the test stopped). However, it did cause a crash.